controllers/search/search.go

```diff
@@ -64,6 +64,10 @@ func SearchHandler(c *gin.Context) {
 	if c.Query("order") == "true" {
 		searchForm.SortOrder = true
 	}
+	searchForm.SortType, err = strconv.Atoi(c.Query("sort"))
```

Comment:
order & sort are not related to the search results nor form but only to the refine form, more precisely only to show the correct value in the two \<select\> after a page load which is just a display thing, so putting it right under the refine bool which is also just a display thing (display refine form yes/no) seems to make sense to me